src/devices/Ads1115/Ads1115.cs

```diff
@@ -0,0 +1,133 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.I2c;
+
+namespace Iot.Device.Ads1115
+{
+    /// <summary>
+    /// Analog-to-Digital Converter ADS1115
+    /// </summary>
+    public class Ads1115 : IDisposable
+    {
+        private I2cDevice _sensor = null;
+
+        private readonly byte _adcMux;
+        private readonly byte _adcPga;
+        private readonly byte _adcRate;
+
+        /// <summary>
+        /// Constructor
+        /// </summary>
+        /// <param name="sensor">I2C Device, like UnixI2cDevice or Windows10I2cDevice</param>
+        /// <param name="mux">Input Multiplexer</param>
+        /// <param name="pga">Programmable Gain Amplifier</param>
+        /// <param name="rate">Data Rate</param>
+        public Ads1115(I2cDevice sensor, InputMultiplexeConfig mux = InputMultiplexeConfig.AIN0, PgaConfig pga = PgaConfig.FS4096, DataRate rate = DataRate.SPS128)
+        {
+            _sensor = sensor;
+            _adcMux = (byte)mux;
+            _adcPga = (byte)pga;
+            _adcRate = (byte)rate;
+
+            Initialize();
+        }
+
+        /// <summary>
+        /// Initialize ADS1115
+        /// </summary>
+        private void Initialize()
+        {
+            byte configHi = (byte)((_adcMux << 4) +
```

Comment:
NIT: It might help if we add a comment on these two bytes that will point to the page of the datasheet where this setup is specified. This is in case we find a bug in the future and need to change the setup, we are then able to tell where did this formula came from.